### PR TITLE
chore(deps): fix dependencies  

### LINKED ISSUE #2034: Why software.amazon.awssdk:dynamodb is not a dependency
### Problem:

In project README you are stating that `software.amazon.awssdk:dynamodb` and `software.amazon.awssdk:kms` are required dependencies. `software.amazon.awssdk:kms` is listed in your POM, but `software.amazon.awssdk:dynamodb` not.

### Solution:

Please add `software.amazon.awssdk:dynamodb` to the POM file, or remove that statement about `DynamoDB` from the README.

### PR DESCRIPTION
**Pull request by rishav-karanjit**
*Issue #, if available:*
closes #2034 

*Description of changes:*
Update pom.xml to include software.amazon.awssdk:dynamodb

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ To use the AWS Encryption SDK for Java you must have:
 
 * **Declare a Dependency on the AWS Encryption SDK in Java and its dependencies**
 
-  This library requires the AWS Cryptographic Material Providers Library in Java from the AWS Java SDK V2.
+  This library requires the AWS Cryptographic Material Providers Library in Java, and the KMS client from the AWS Java SDK V2.
```
Comment (texastony) [inline README.md:26]: Revert this.
See line 30, which covers this requirement.

File: pom.xml [hunk 2]
```diff
@@ -59,6 +59,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>kms</artifactId>
+            <version>2.20.91</version>
```
Comment (texastony) [inline pom.xml:65]: You do not need to declare a version here.
Read https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
In particular, expand the POM.
```suggestion
```

File: README.md
Comment (texastony) [inline README.md:1]: The README still has a mistake for the AWS SDK V1's KMS Version.

**Review by texastony — CHANGES_REQUESTED** [review]

**Comment by rishav-karanjit** [comment]
The readme needs more fixes then just DDB

**Pull request closed without merging**